string bot_name
---
int64 goal_x
int64 goal_y
time stamp
bool success